meta-ostro/recipes-swupd/swupd-client/swupd-client/efi_combo_updater.c

```diff
@@ -0,0 +1,136 @@
+#include <stdio.h>
```

Comment:
This whole file should be rewritten, but that's something for a different PR. For now, let's use it as it is.

The reason for rewriting:
- was meant to remove a dependency on /bin/sh, but now still uses popen() and thus still depends on /bin/sh
- assert() for error checking